### PR TITLE
Reviewing the User Documentation

### PR DESCRIPTION
**Pull request by michalmilc90**
topic reviewed: "Content Model"

### CODE REVIEW

File: docs/content_model.md [hunk 1]
```diff
@@ -89,26 +89,27 @@ They represent three categories of Content Types that you can create
...
 
+User accounts, which are also considered content in eZ Platform, are viewable in the **Users** tab accessible in the **Admin Panel** (see [Users](organizing_the_site.md#users)).
+
 During your work with eZ Platform you will often be asked to select content from the repository.
-This happens for example when you want to move or copy, a Content item, embed an image, link two Content items, etc.
+This happens for example when you want to move or copy a Content item, embed an image, link two Content items, etc.
 In such cases you will make use of the **Content Browser**.
 
-The Content Browser enables you to navigate through all content in the site, including regular content, media and User accounts.
+The **Content Browser** allows you to navigate through all content in the site, including regular content, media, and User accounts.
```
Comment (DominikaK) [inline docs/content_model.md]: We prefer to use "enables" in this context, while "allows" is reserved for talking about permissions in the system.

File: docs/creating_content_advanced.md [hunk 2]
```diff
@@ -51,10 +50,10 @@ SiteAccess must be [set up by the administrator of the installation](https://doc
 
     ## Forms
 
-    Forms enable you to place a survey, questionnaire, sign-up form, etc. on your site.
+    Forms allow you to place a survey, questionnaire, sign-up form, etc. on your website.
```
Comment (DominikaK) [inline docs/creating_content_advanced.md]: as above

File: docs/creating_content_advanced.md [hunk 3]
```diff
@@ -86,11 +85,11 @@ SiteAccess must be [set up by the administrator of the installation](https://doc
...
     ### Viewing results
 
-    Once you publish a form and users start filling it in, you can preview the results in the Submissions tab in Content item view.
+    Once you publish a form and users start filling it in, you can preview the results in the **Submissions** tab in the Content item view.
 
     Here you can view the details of each submission.
     You can also delete any submissions (for example if they were made while testing or contain spam).
 
-    The Download Submissions button enables you to download all the submissions in a CSV (comma-separated value) file.
+    The **Download Submissions** button allows you to download all the submissions in a .CSV (comma-separated value) file.
```
Comment (DominikaK) [inline docs/creating_content_advanced.md]: as above

File: docs/organizing_the_content.md [hunk 4]
```diff
@@ -97,42 +97,42 @@ You can also permanently remove all content from the Trash by clicking **Empty T
...
-You can set up Sections in the Admin Panel in the **Sections** tab.
+You can set up Sections in the **Admin Panel**, in the **Sections** tab.
 A few Sections are provided with each installation, and you can add new ones.
 
 ![List of Sections](img/sections.png "List of Sections")
 
-In the Sections tab you can assign Sections to content or delete them.
-You can only delete Sections that don't contain any content.
+In the **Sections** tab, you can assign Sections to content or delete them.
+You can only delete Sections that don not contain any content.
```
Comment (DominikaK) [inline docs/organizing_the_content.md]:
```suggestion
You can only delete Sections that do not contain any content.
```

File: docs/creating_content_advanced.md [hunk 5]
```diff
@@ -1,45 +1,44 @@
-# Creating content, advanced
+# Creating content – advanced
 
 ## Languages
 
 The content on your website can be translated into different languages. Each Content item can have different language versions.
-Which version is shown to a visitor depends on the way your installation is set up (see [SiteAccess](#siteaccess)).
+The version visible to a visitor depends on the way your installation is set-up (see [SiteAccess](#siteaccess)).
```
Comment (MagdalenaZuba) [inline docs/creating_content_advanced.md]:
```suggestion
The version visible to a visitor depends on the way your installation is set up (see [SiteAccess](#siteaccess)).
```

File: docs/creating_content_advanced.md [hunk 6]
```diff
@@ -1,45 +1,44 @@
...
-After adding a language you may have to reload the application to be able to use it.
+Every new language must have a name and a language code written in the xxx-XX format, for example eng-US, fre-FR, nor-NO, etc.
+After adding a language, you may have to reload the application to be able to use it.
 
 !!! caution
 
-    After adding a language you should be able to start adding translations to your content.
-    However, depending on the way your website it set up, additional configuration will probably be necessary
+    After adding a language, you should be able to start adding translations to your content.
+    However, depending on the way your website is set-up, additional configuration may be necessary
```
Comment (MagdalenaZuba) [inline docs/creating_content_advanced.md]:
```suggestion
    However, depending on the way your website is set up, additional configuration may be necessary
```

File: docs/creating_content_basic.md [hunk 7]
```diff
@@ -1,38 +1,39 @@
...
 
-1. Click **Create** in the right menu *whenever you are viewing any other content*.
-The new item will be placed under the current Content item in the tree.
-1. Click **Create** in the upper right corner of the *Dashboard*.
+1. Click **Create** in the right menu while viewing any other content.
+The new item appears under the current Content item in the tree.
+1. Click **Create** in the upper-right corner of the Dashboard.
 In this case you have to select where the content will be located.
-1. Go to the **Create** tab in the *Universal Discovery Widget*.
+1. Go to the **Create** tab in the **Universal Discovery Widget**.
```
Comment (MagdalenaZuba) [inline docs/creating_content_basic.md]:
```suggestion
1. Go to the **Create** tab in the *Universal Discovery Widget*.
```
 bold is reserved for names that are visible on the page or in the Back Office

File: docs/creating_content_basic.md [hunk 8]
```diff
@@ -1,38 +1,39 @@
...
 Here also you have to choose a location for the new item.
-1. Drag files onto the *Sub-items box* when viewing any content or click the upload button and select files to upload.
-In this way you can upload files such as images, videos, pdf documents etc.
+1. Drag files onto the **Sub-items box** when viewing any content or click the upload button and select files to upload.
```
Comment (MagdalenaZuba) [inline docs/creating_content_basic.md]:
```suggestion
1. Drag files onto the **Sub-items** box when viewing any content or click the upload button and select files to upload.
```

File: docs/faq.md [hunk 9]
```diff
@@ -4,27 +4,27 @@
 
 ### Product
 
-**What is the difference between eZ Platform and Enterprise Edition?**
+**What is the difference between eZ Platform and eZ Enterprise?**
```
Comment (MagdalenaZuba) [inline docs/faq.md]: Official name is eZ Platform Enterprise Edition, so I would leave `Enterprise Edition` as it is or change to `eZ Platform Enterprise Edition`.

File: docs/faq.md [hunk 10]
```diff
@@ -4,27 +4,27 @@
...
 
-eZ Platform is open-source. Enterprise Edition bases on eZ Platform and adds new functionalities.
+eZ Platform is open-source. eZ Enterprise bases on eZ Platform, but features new functionalities.
 
-**Which parts of eZ are open source and which are commercial?**
+**Which parts of eZ are open-source and which are commercial?**
```
Comment (MagdalenaZuba) [inline docs/faq.md]:
```suggestion
**Which parts of eZ are open source and which are commercial?**
```
`open source` is more common in our documentation. If you want to change that, let's discuss and add to terminology :)

File: docs/organizing_the_site.md [hunk 11]
```diff
@@ -32,34 +32,34 @@ To delete Field(s), check the boxes next to any Field names and select the trash
 
 When a Content Type is modified, each instance of this type (each Content item based on this Content Type) will be changed as well.
 If a new Field is added to a Content Type, this Field will appear in every relevant Content item.
-Therefore, if the new Field is required, it's recommended to provide a default value for it.
+Therefore, if the new Field is required, it is recommended to provide a default value for it.
 If a Field is deleted from the Content Type definition, all these Fields will be removed from Content items of this type.
 
 ![Modifying Content Type message](img/notification_ct.png "Modifying Content Type message")
 
 Each Field has a number of common parameters:
```
Comment (MagdalenaZuba) [inline docs/organizing_the_site.md]: A table would look nice in this place :)

**Pull request merged**